exchanges/ftx/ftx_test.go

```diff
@@ -356,6 +383,27 @@ func TestTriggerOrder(t *testing.T) {
 	}
 }
 
+func TestCancelOrder(t *testing.T) {
```

Comment:
can CancelOrder account for using ClientOrderID too? during my implementation i didnt add it, but can we have an if statement to see if ClientOrderID != ""  n then use DeleteOrderByClientID

Reply:
Yep!